test/features/support/env.rb

```diff
@@ -28,3 +27,9 @@
 World(OllertTest::Navigation) do
   Ollert.new
 end
+
+Trello.configure do |config|
+  config.developer_public_key = ENV['INTEGRATION_KEY']
```

Comment:
This is kind of strange because the tests run the application which uses the  PUBLIC_KEY variable to access Trello but then the tests themselves are using this INTEGRATION_KEY variable.